tasks/sep-dev-0/004-fp-granite/SignFromJson.s.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+
+contract NestedSignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    DisputeGameFactory dgfProxy = DisputeGameFactory(0x2419423C72998eb1c6c15A235de2f112f8E38efF);
+
+    // Safe contract for this task.
+    address immutable proxyAdminOwnerSafe = vm.envAddress("OWNER_SAFE");
+    address immutable livenessGuard = 0x24424336F04440b1c28685a38303aC33C9D14a25;
```

Comment:
Not sure where this value is documented?  The `isModuleEnabled()` [method on the OWNER_SAFE](https://sepolia.etherscan.io/address/0x4377BB0F0103992b31eC12b4d796a8687B8dC8E9#readProxyContract#F13) is returning false.

Reply:
Left this in given it's just in the list of allowed storage access. It is referenced in other tasks - I think its actually the mainnet liveness guard so doesn't actually apply here.